buildSrc/src/main/kotlin/com/datadog/gradle/plugin/srpayloadupdate/SrPayloadCreateTask.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.gradle.plugin.srpayloadupdate
+
+import org.gradle.api.tasks.Exec
+
+open class SrPayloadCreateTask : Exec() {
```

Comment:
why this class should be `open`?

Reply:
that's the request for a gradle task class